host/src/components/HeaderGame.jsx

```diff
@@ -46,14 +46,14 @@ const useStyles = makeStyles(() => ({
 }));
```

Comment:
let's revert any changes here so as to remove this file from the PR. We should only be including `play` files directly relating to the timer refresh issue